plugins/org.jkiss.dbeaver.ext.generic/src/org/jkiss/dbeaver/ext/generic/GenericConstants.java

```diff
@@ -45,6 +45,8 @@ public class GenericConstants {
     public static final String PARAM_OMIT_SINGLE_CATALOG = "omit-single-catalog";
     public static final String PARAM_OMIT_SINGLE_SCHEMA = "omit-single-schema";
     public static final String PARAM_OMIT_CATALOG_NAME = "omit-catalog-name";
+    public static final String PARAM_STORES_UNQUOTED_CASE = "stores-unquoted-case";
+    public static final String PARAM_STORES_QUOTED_CASE = "stores-quoted-case";

```

Comment:
They should be called `stored-xxx` instead of `stores-xxx`, actually. Despite the same naming in the `SQLDialect` class, it's clearly a typo.